dependency-charts/datapusher/README.md

```diff
@@ -13,8 +13,15 @@ Current chart version is `1.0.0`
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
+| datapusher.chunkSize | string | `"16384"` | Size of chunks of the data that is being downloaded in bytes |
+| datapusher.datapusherRewriteResources | string | `"True"` | Enable or disable (boolean) whether datapusher should rewrite resources uploaded to CKAN's filestore, since datapusher takes the CKAN Site URL value for generating the resource URL. Default: False |
+| datapusher.datapusherRewriteUrl | string | `"http://ckan"` |  |
+| datapusher.datapusherSslVerify | string | `"False"` | Enable or disable (boolean) verification of SSL when trying to get resources. Default: True |
+| datapusher.downloadTimeout | string | `"30"` | Timeout limit of the download request |
+| datapusher.insertRows | string | `"250"` | Number of rows to take from the data and upload them as chunks to datastore |
+| datapusher.maxContentLength | string | `"10485760"` | Maximum size of content to be uploaded in bytes. |
 | fullnameOverride | string | `""` |  |
-| image.pullPolicy | string | `"IfNotPresent"` |  |
+| image.pullPolicy | string | `"Always"` |  |
```

Comment:
Now the default is `IfNotPresent`. I think once this is updated we're good to go.